test/testcafe/framework/page-objects/EnrollEmailPageObject.js

```diff
@@ -21,12 +22,55 @@ export default class EnrollAuthenticatorPhonePageObject extends BasePageObject {
   }
 
   async enterCodeFromEmailLinkExists() {
-    const elCount = await Selector(ENTER_CODE_FROM_EMAIL).count;
-    return elCount === 1;
+    if (userVariables.v3) {
+      const elCount = await Selector('[type="button"]').count;
+      return elCount === 1;
+    } else {
+      const elCount = await Selector(ENTER_CODE_FROM_EMAIL_CLASS).count;
+      return elCount === 1;
+    }
   }
 
   async clickElement(selector) {
     await this.form.clickElement(selector);
   }
 
+  async clickEnterCodeInstead() {
+    if (userVariables.v3) {
+      await this.form.clickButton(ENTER_CODE_FROM_EMAIL_TEXT);
+    } else {
+      await this.form.clickElement(ENTER_CODE_FROM_EMAIL_CLASS);
+    }
+  }
+
+  async resendEmailExists() {
+    if (userVariables.v3) {
+      return Selector('[role="alert"]').exists;
```

Comment:
We should use one of these existing selectors (see `BaseFormObject`):
* `this.form.getErrorBoxCount`
* `this.form.getErrorBoxText`
* `this.form.hasErrorBox`